projectq/ops/_basics.py

```diff
@@ -262,12 +272,19 @@ def __eq__(self, other):
             return True
         return False
 
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
```

Comment:
Why is that necessary? Isn't `__ne__()` with the same implementation inherited from `BasicGate`?

Reply:
Sorry my bad, of course it uses `__ne__` from BasicGate. 
(It's a habit from Python2's old classes)